.travis.yml

```diff
@@ -0,0 +1,36 @@
+sudo: required
+language: rust
+rust:
+  - nightly
+  - beta
+  - stable
```

Comment:
For nightly we should allow failures https://github.com/sdroege/gobject-subclass/blob/1f0334f275b8f5e987f9b57474c5cf1783665eda/.travis.yml#L16-L18